src/ngraph/runtime/intelgpu/intelgpu_backend.cpp

```diff
@@ -276,15 +280,46 @@ extern "C" void delete_backend(runtime::Backend* backend)
     delete backend;
 }
 
+static size_t get_max_memory_rss_in_kb()
```

Comment:
The system call return it in kilobytes. It means 1000 bytes. In case of 1024 it called KiB (kibibytes) by the standard.
According to this http://man7.org/linux/man-pages/man2/getrusage.2.html
```
       ru_maxrss (since Linux 2.6.32)
              This is the maximum resident set size used (in kilobytes).
              For RUSAGE_CHILDREN, this is the resident set size of the
              largest child, not the maximum resident set size of the
              process tree.
```

Reply:
So why not return bytes? Just multiply by 1000. Many/most uses of k in computers is interpreted as 1024. Since it can be misinterpreted remove the chance for it to be misinterpreted. I don't want to find documentation on ru_maxrss to find out what the return is.